src/sentry/issues/auto_source_code_config/stacktraces.py

```diff
@@ -19,6 +19,8 @@ def get_frames_to_process(data: NodeData | dict[str, Any], platform: str) -> lis
     for stacktrace in stacktraces:
         frames = stacktrace["frames"] or []
         for frame in frames:
+            if frame is None:
```

Comment:
Ideally `get_stacktraces` would not allow returning invalid date or even `get_path` to do a better job of filtering out Nones.